src/olympia/lib/settings_base.py

```diff
@@ -1121,9 +1144,6 @@ def JINJA_CONFIG():
         'queue': 'addons'},
     'olympia.addons.tasks.version_changed': {'queue': 'addons'},
 
-    # Files (goes to devhub queue).
-    'olympia.files.tasks.update_webext_descriptions_all': {'queue': 'devhub'},
```

Comment:
We need this to be missing from `CELERY_ROUTES` to check that setting `CELERY_QUEUES` fixes the duplicate task problem... but at the same time it should (ideally) be defined.